common/wforce-webserver.cc

```diff
@@ -224,17 +224,21 @@ void WforceWebserver::connectionThread(WforceWebserver* wws)
         // set the defaults in case we don't find a command
         resp.status = 404;
         resp.body = R"({"status":"failure", "reason":"Command not found"})";
-    
+
+        std::string new_content_type;
```

Comment:
totally unrelated change.

Reply:
Why is this unrelated? The purpose of this new function is to be able to return different content types from the GET functions. Not also allowing that for the other types of functions would be reverse

Reply:
i cannot see how it's related to the commit, so perhaps you should do this as a preparation before this commit?

Reply:
ok